### PR TITLE
CHANGELOG for 2019.01

### PR DESCRIPTION
**Pull request by tobiasd**
Please have a look for spelling stuff and whether I forget anything crucial.

### CODE REVIEW

**Comment by AlfredSK** [comment]
Looking good. Found no obvious errors.

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,88 @@
...
+    Fixed a bug that caused some photo albums having no permissions [JonnyTischbein]
+    Fixed a problem adding multiple hashtags [JonnyTischbein]
+    Fixed a bug when subscribing to OStatus accounts [MrPetovan]
+    Fixed a bug in SQL grammar [Alkarex]
+    Fixed a bug with WebSub [Alkarex]
+    Fixed a bug in the generated node info JSON [fabrixxm]
+    Moved config format to PHP arrays [MrPetovan]
+    Moved several additional features back to the default features [annando]
+    Started to deprecate the Google+ support [annando]
+    Added support of ActivityPub (tested with Mastodon, Nextcloud Social, PeerTube, Pleroma) [annando]
```
Comment (annando) [inline CHANGELOG]: And Pixelfed.

**Comment by annando** [comment]
Okay, when do we do the release?

**Comment by tobiasd** [comment]
I'm working on it ;-) 

How long is the downtime you expect from the DB changes?

**Comment by MrPetovan** [comment]
Looks good. Ship it!

**Comment by JeroenED** [comment]
I think I don't see this one?

https://github.com/friendica/friendica/pull/5966

**Pull request merged**